tutorials/quick_start/keras_fw/keras_applications/model_lib_keras_applications.py

```diff
@@ -0,0 +1,156 @@
+            raise Exception(f'model {args[MODEL_NAME]} has a dynamic input size, which is not supported by the MCT')
+        super().__init__(args)
+
+    def get_keras_apps_model(self, model_name: str):
+        """
+        Extracts the model package and model name from the input string: <package>.<model>
+        The model package class contains the 'preprocess_input' method for preprocessing input images
+        One exception is the MobileNetV3 models that are located in the parent model (tf.keras.applications)
+
+        Args:
+            model_name (str): A string containing the model to quantize in the following format: <package>.<model> (e.g. mobilenet_v2.MobileNetV2)
+
+        Returns:
+            model package class
+            model class
+
+        """
+        model_attrs = model_name.split('.')
+        if len(model_attrs) == 1 and model_attrs[0] in self.MISPLACED_CLASSES:
+            # the MobNetV3 models aren't located in their own package as other models, but rather in the general package
+            model_package, model_class = 'mobilenet_v3', model_attrs[0]
+        elif len(model_attrs) != 2:
+            raise Exception(f'Keras Applications expects model_name of format <model_package>.<model_class>, but got {model_name}')
+        else:
+            model_package, model_class = model_attrs
+        if hasattr(tf.keras.applications, model_package):
+            model_package = getattr(tf.keras.applications, model_package)
+            if model_class in self.MISPLACED_CLASSES:
+                return getattr(tf.keras.applications, model_class), model_package
+            elif hasattr(model_package, model_class):
+                return getattr(model_package, model_class), model_package
+            else:
+                raise Exception(f'Unknown Keras Applications model class {model_class}')
+        else:
+            raise Exception(f'Unknown Keras Applications model package {model_package}, Please check available models in https://www.tensorflow.org/api_docs/python/tf/keras/applications')
+
+    @staticmethod
+    def get_keras_apps_weights(model_name):
+        return None
+
+    def generate_preprocess(self, truncate_preprocess: bool = False):
+        """
+        Generates the preprocess function for evaluation and quantization (representative dataset)
+        Some models in this library contain the normalization of the preprocess in the beginning of the model. They can be removed.
+        Args:
+            truncate_preprocess (bool): Removes Normalization & Rescaling layers from the beginning of the model
+
+        Returns:
+
+        """
+        pp_model = None
+        if truncate_preprocess:
+            self.model, pp_model = separate_preprocess_model(self.model)
+
+        def _preprocess(x):
+            x = self.model_package.preprocess_input(x)
+            if pp_model is not None:
+                x = pp_model(x)
+            return x
+
+        return _preprocess
```

Comment:
why not to return the function directly? 
return self.model_package.preprocess_input(x)
I would try to avoid extra function definitions...